### PR TITLE
Feature/string enrichments val syntax

### PR DESCRIPTION
**Pull request by moltude**

### CODE REVIEW

**Comment by mdellabitta** [comment]
The Codacy failure for this is annoying because it's fine with the other `type` definition right after it: https://www.codacy.com/app/dpla/ingestion3/pullRequest?bid=4821746&prid=738018&utm_campaign=new_commit&utm_medium=Email&utm_source=Internal&

**Comment by markbreedlove** [comment]
@AudreyAltman @moltude @mdellabitta: Anyone else want to give a review before this is merged?

If we're ready, one of us three committers should squash the changes.

File: src/main/scala/dpla/ingestion3/enrichments/StringEnrichments.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  val convertToSentenceCase: SingleStringEnrichment = (value) => {
+    val pattern: Regex = """.*?(\.)""".r
+    val sentences = for( t <- pattern findAllIn value) yield t.trim.capitalize
+    // rejoin the sentences and add back the whitespace that was trimmed off
+    sentences.mkString(" ")
+  }
+
+  val limitCharacters: SingleStringEnrichment = (value) => {
+    ""
+  }
```
Comment (AudreyAltman) [inline src/main/scala/dpla/ingestion3/enrichments/StringEnrichments.scala]: I'm not sure I understand the use case for `limitCharacters`, but that could be my unfamiliarity with the ingestion process.  In any case, it seems to be the only method without a test.  Does it need one?

Comment (mdellabitta) [inline src/main/scala/dpla/ingestion3/enrichments/StringEnrichments.scala]: It's basically for gargantuan titles as far as I know. We should probably test it.

Comment (mdellabitta) [inline src/main/scala/dpla/ingestion3/enrichments/StringEnrichments.scala]: Oh, the feature itself is not implemented. I guess we thought it was so easy, we could get to it later.

Comment (AudreyAltman) [inline src/main/scala/dpla/ingestion3/enrichments/StringEnrichments.scala]: Maybe add a `TODO` comment to clarify?

**Comment by AudreyAltman** [comment]
The `limitCharacters` implementation looks good!  Thanks @mdellabitta

**Comment by mdellabitta** [comment]
All those years of programming have been leading up to this moment! :)

**Comment by AudreyAltman** [comment]
Rebase to develop?

**Comment by mdellabitta** [comment]
Rebased on develop + squashed.

**Pull request merged**